src/main/resources/entity-types/inventory/simple_service_point_details.json5

```diff
@@ -140,6 +140,8 @@
           label: 'False',
         },
       ],
+      filterValueGetter: "\"left\"(lower(:sourceAlias.jsonb ->> 'pickupLocation'::text), 600)",
+      valueFunction: "\"left\"(lower(:value), 600)"
```

Comment:
Nit: Can you put these next to the `valueGetter`? Having `valueGetter`, `filterValueGetter`, and `valueFunction` grouped together helps with readability, since the 3 are closely related